images/ubuntu/templates/ubuntu-22.04.pkr.hcl

```diff
@@ -138,6 +138,81 @@ variable "vm_size" {
   default = "Standard_D4s_v4"
 }
 
+variable "sources" {
+  type    = list(string)
+  default = ["source.azure-arm.build_image"]
```

Comment:
input as variable with default value for backward compatibility reasons
azure is always build as default if no sources variable is provided